anaconda.spec.in

```diff
@@ -6,7 +6,7 @@ License: GPLv2+ and MIT
 URL:     http://fedoraproject.org/wiki/Anaconda
 
 # This should should only be set for development purposes for the time
-%global use_cockpit 1
```

Comment:
Is this needed for something else aside for the releases? For the release we let the seddery in the packit change this.

Reply:
With this `container-rpms` will fail. In general, we don't use packit everywhere. Also there is no reason to have this enabled here.

Reply:
The benefit of this is that it doesn't need to be reverted on master. I like that. On the other hand, we have to revert plenty of other stuff anyway so it's not that much work at the end. And also we will probably soon remove that on master anyway because there is no reason to have it there.